nixos/lib/testing-python.nix

```diff
@@ -186,6 +186,12 @@ rec {
           --set startScripts "''${vmStartScripts[*]}" \
           --set testScript "$out/test-script" \
           --set vlans '${toString vlans}'
+
+        ln -s ${testDriver}/bin/nixos-test-driver $out/bin/nixos-run-vms
+        wrapProgram $out/bin/nixos-run-vms \
+          --set startScripts "''${vmStartScripts[*]}" \
+          --set testScript "${pkgs.writeText "start-all" "start_all(); join_all();"}" \
+          --set vlans '${toString vlans}'
```

Comment:
would it be possible to render this somwhere else, instead? I'd not be totally unhappy, if we could keep this entrypoint here minimal, so that we ca evolve a (single) consistent api, here.

Reply:
Not sure if I'm misunderstanding you, but since it's a symlink to the test-driver it's (more or less) a wrapper already, correct?

Do you have a better place in mind already? E.g. only adding it to `$out` if `nixos-build-vms` is used rather than an "actual" VM test? Not sure if I like that (for consistency reasons), but I'd like to understand what you have in mind for now :)

Reply:
> E.g. only adding it to $out if nixos-build-vms is used rather than an "actual" VM test?

Oh, of that was possible, I would indeed favor it.

The reason is that I think we shouldn't broadly advertise an interface through its existence in a place where I have a feeling "less is more".

As the author of this refacor I got a pretty good sense of the situations we can get in if there is a "convolution of interfaces".

So I'm just worried, in principle, that if it's here (instead of there), them over the eons of times undesired ideas might sneak in again and slowly destructure the code into minestrone soup (again).

On the other hand, calling site interfaces ("$out if nixos-build-vms") are a beautiful thing, to avoid this.

If I couldn't convince you, I'm fine, but if I could, I'm happy too. :smile:

Reply:
Since I use `nixos-build-vms(8)` rather frequently as a maintainer to reproduce bugs or to test modules, I have a fairly strong opinion about it, but the test-driver itself can work without `nixos-run-vms` as well (in other words: no strong opinion about).

I'll check today if there's a reasonably trivial way to implement this and if yes, I'd just make that change.

Reply:
Updated the PR :) 
Is the removal from the API surface sufficient for you then?

Reply:
this is an actual improvement! nice! yep, that's completely fine.